lib/excerpt_parser.rb

```diff
@@ -4,7 +4,7 @@ class ExcerptParser < Nokogiri::XML::SAX::Document
 
   attr_reader :excerpt
 
-  SPAN_REGEX = /<\s*span[^>]*class\s*=\s*['|"]excerpt['|"][^>]*>/
+  CUSTOM_EXCERPT_REGEX = /<\s*[span|div][^>]*class\s*=\s*['|"]excerpt['|"][^>]*>/
```

Comment:
The regex is not correct. And wasn't correct before. 😛 It confuses character sets and capturing groups:

```suggestion
  CUSTOM_EXCERPT_REGEX = /<\s*(span|div)[^>]*class\s*=\s*['"]excerpt['"][^>]*>/
```

Reply:
A perfect catch 👍